sormas-ui/src/main/java/de/symeda/sormas/ui/person/LocationsField.java

```diff
@@ -40,6 +43,18 @@ protected void editEntry(LocationDto entry, boolean create, Consumer<LocationDto
 			entry.setUuid(DataHelper.createUuid());
 		}
 
+		if (create) {
+			List<CountryReferenceDto> countryItems = FacadeProvider.getCountryFacade().getAllActiveAsReference();
+			CountryReferenceDto serverCountryDto = FacadeProvider.getCountryFacade().getServerCountry();
+			CountryReferenceDto defaultCountry = countryItems.stream()
+				.filter(
+					countryReferenceDto -> serverCountryDto != null
+						&& countryReferenceDto.getIsoCode().equalsIgnoreCase(serverCountryDto.getIsoCode()))
```

Comment:
Replace ISO code with name

Reply:
See first part of https://github.com/hzi-braunschweig/SORMAS-Project/pull/4336#discussion_r576855183